tests/scanner/audit/iam_rules_engine_test.py

```diff
@@ -1405,6 +1403,87 @@ def test_project_required(self):
         ])
         self.assertItemsEqual(expected_violations, actual_violations)
 
+    def test_policy_all_projects_must_have_owners_from_domain(self):
```

Comment:
Please clarify this method name, to show that it's using the domain member type.  So, a good name would be:
```
test_policy_all_projects_must_have_owners_from_domain_type
```